config/saml.yml

```diff
@@ -0,0 +1,13 @@
+development: &defaults
+  issuer: 'saml-rp.vetsgov.localhost'
+  callback_url: 'http://localhost:3000/auth/saml/callback'
+  metadata_url: 'https://api.idmelabs.com/saml/metadata'
+
+
+test:
+  <<: *defaults
+
+production:
+  issuer: <%= ENV["SAML_ISSUER"] %>
+  callback_url: <%=ENV["CALLBACK_URL"] %>
+  metadata_url: <%= ENV["METADATA_URL"] %>
```

Comment:
This doesn't seem to work with environment variables set in application.yml, unfortunately :(